### PR TITLE
[MRG] Add sliding window to benchmark

### PR DESCRIPTION
**Pull request by rflamary**
Positive activation:
image

Non positive activations:
image

### CODE REVIEW

File: solvers/sw_csc.py [hunk 1]
```diff
@@ -0,0 +1,1023 @@
...
+        return np.reshape(
+            self.w, (self.D.shape[2], self.y.shape[1], 1)
+        )[:, :-self.D.shape[1]+1, :]
+
+
+fmt_verb = '| {:4d} | {:4d} | {:1.5e} |'
+fmt_verb2 = '| {:4s} | {:4s} | {:11s} |'
+
+
+def solve_lasso(y, H, a0, lambd, tol=1e-4, mode="C", verbose=False, solver='celer', positive=False):
```
Comment (tomMoral) [inline solvers/sw_csc.py]:
```suggestion
def solve_lasso(y, H, a0, lambd, tol=1e-4, mode="C", verbose=False, solver='celer',
                positive=False):
```

**Comment by rflamary** [comment]
on it for clearing spams correctly

**Comment by tomMoral** [comment]
Oups, just did it.

**Comment by rflamary** [comment]
it' OK ;)

**Review by tomMoral — APPROVED** [review]
Awesome! LGTM thx @rflamary !! :)

**Pull request merged**